js/inventory.js

```diff
@@ -0,0 +1,42 @@
+let isInventoryOpen = false;
+
+document.querySelector(".inventorySlot.slotArrow").onclick = function() {
+    if (!isInventoryOpen) {
+        moveInventory("-", 85, 0, 0);
+        isInventoryOpen = true;
+        document.querySelector(".blackoutPanel").classList.remove("hidden");
+    } else if (isInventoryOpen) {
+        moveInventory("+", 40, 180, 0.5);
+        isInventoryOpen = false;
+        setTimeout(() => {
+            document.querySelector(".blackoutPanel").classList.add("hidden");
+        }, 400)
+    }
+}
+
+function moveInventory(direction, startVh, startDeg, startOpacity) {
+    const element = document.querySelector(".playerInventory");
+    const endVh = direction === "+" ? startVh + 45 : startVh - 45;
+    const duration = 300; // ms
+    const startTime = performance.now();
+    const endDeg =  direction === "+" ? startDeg + 180 : startDeg - 180;
+    const endOpacity = direction === "+" ? startOpacity - 0.5 : startOpacity + 0.5;
```

Comment:
What do u think about this order?
```suggestion
    const element = document.querySelector(".playerInventory");
    
    const duration = 300; // ms
    const startTime = performance.now();
    
    const endVh = direction === "+" ? startVh + 45 : startVh - 45;
    const endDeg =  direction === "+" ? startDeg + 180 : startDeg - 180;
    const endOpacity = direction === "+" ? startOpacity - 0.5 : startOpacity + 0.5;
```